src/core.js

```diff
@@ -552,7 +552,7 @@ $.extend( $.validator, {
 			var count = 0,
 				i;
 			for ( i in obj ) {
-				if ( obj[ i ] ) {
+				if ( obj[ i ] !== undefined && obj[ i ] !== null && obj[ i ] !== false ) {
```

Comment:
Please add a comment that we do this verbose check for the empty string exception